File: factorial/factorial.py
from datetime import datetime
import sys

def ts():
    return datetime.now().timestamp()

def execution_time(start: datetime.timestamp, end: datetime.timestamp) -> None:
    print(f"{round((end - start) * 1000, 3)} ms")

def factorial(n: float):
    if n < 0:
        return None
    return n * factorial(n - 1) if n > 0 else 1

if __name__ == "__main__":
    start = ts()

    for _ in range(10**5):
        factorial(25)

    end = ts()
    execution_time(start, end)

File: factorial/factorial.js
const {performance} = require('perf_hooks');
const ts = () => { return performance.now() }
const execution_time = (start, end) => { console.log(`${(end - start).toFixed(3)} ms`) }

const factorial = (n) => {
    if (n < 0) return Number.NaN
    if (n === 0) return 1

    return n * factorial(n - 1)
}

let st = ts()

for (let i = 0; i < 100000; i++)
    factorial(25)

let en = ts()
execution_time(st, en)